tests/ui/numbered_fields.fixed

```diff
@@ -34,4 +34,8 @@ fn main() {
 
     // Aliases can't be tuple constructed #8638
     let _ = Alias { 0: 0, 1: 1, 2: 2 };
+
+    struct TupleStructVec(Vec<usize>);
```

Comment:
nit: please mention that this is  for #12367 in a comment (and then regen the test expectations)